src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -805,7 +805,7 @@ private long newestVersion(DHTKey key) {
 
     private void handleTriggeredWaitFors(Set<Waiter> triggeredWaitFors) {
         if (debugWaitFor) {
-            System.out.println("handleTriggeredWaitFors");
+            Log.fineAsync("handleTriggeredWaitFors");
```

Comment:
debugWaitFor is off by default. Make Log.warningAsyncf so that the debugWaitFor flag has an effect